includes/class-activator.php

```diff
@@ -23,16 +23,42 @@
 class WI_Volunteer_Management_Activator {
 
 	/**
-	 * On activation flush rewrite rules.
+	 * Run the necessary activation process for all sites or the current one.
 	 *
-	 * On activation we first declare our custom post type and then
-	 * flush rewrite rules so our custom url structure works how we'd want it to.
-	 * Along with that we create our volunteer role to be used to easily track volunteers
-	 * who sign up for volunteer opportunities on the website.
+	 * If we're working with a multisite installation and the plugin is being
+	 * turned on network-wide then run the activation on all sites in the network.
+	 * If not, run the activation only on the current site.
 	 *
 	 * @since    0.1
+	 * @param 	 $network_wide Whether the plugin is being network enabled on multisite
 	 */
-	public static function activate() {
+	public static function activate( $network_wide = false ) {
```

Comment:
Nice job coming up with this solution! Looks and works great.

Reply:
Thank you good sir.